SillyBusinessInc/SillyBusinessGame
# Feat: #37 replace whole input system

## Purpose of this PR:
i replaced the whole input system instead of using key pressed or keydown E. instead of that i am using a category like if move is pressed or being hold or something.

## How to test:
All the inputs of the player has changed so you can test it out in the scene. To see what input has changed check the InputSystem_Actions file in URP map or somewhere in the settings not exactly sure where. 

if you can't find the keys check:
wasd/arrow keys/ joystick for moving
space hold for glide
ctrl for dodge
space for jump

### links: 
[Ticket](https://github.com/SillyBusinessInc/SillyBusinessGame/tree/feat/%2337-Replace-whole-input-system)  
[Documentation link](https://hrnl.sharepoint.com/:w:/r/sites/GameDesignDevelopment/_layouts/15/Doc.aspx?sourcedoc=%7BE7AE02C4-3CF2-49E8-9A49-653A1A85DC97%7D&file=Technical%20Documentation.docx&action=default&mobileredirect=true)
